ui/app/src/views/PegListingPage.vue

```diff
@@ -111,14 +111,14 @@ export default defineComponent({
       searchText,
       peggedSymbol(unpeggedSymbol: string) {
         if (unpeggedSymbol.toLowerCase() === "erowan") {
-          return "rowan";
+          return "ROWAN";
```

Comment:
Will this cause problems as these set the route symbol? Maybe it is not as common these days but I would have thought it best to keep everything lowercase and parse on input incase certain browsers and OSs treat url casing differently.

Reply:
This does effect routing. Not sure if breaking. But does make me nervous
<img width="451" alt="Screen Shot 2021-02-22 at 2 59 35 PM" src="https://user-images.githubusercontent.com/5235538/108781605-039d2b80-751f-11eb-883b-4c42dba321ce.png">